source/Use-Protocol.rst

```diff
@@ -23,15 +26,25 @@ Requests
 --------
 The requests exchange is used for round-trip information transfers in the form of T_Request messages and their resulting T_Reply messages. Routing keys for requests begin with the name of the endpoint being targeted (or broadcast, for that [special case described below](#broadcast-requests)), and may include a routing key specifier to indicate a particular command (for an OP_CMD) or attribute (for an OP_GET or OP_SET which configures an endpoint rather than assigning or querying the endpoint itself). The T_REPLY is then sent to the routing key specified in the properties provided along with the AMQP message.
 
+Properties
+----------
+
+* Type: ``EXCHANGE_TYPE_DIRECT``
+* Passive: ``false``
+* Durable: ``false``
+* Auto Delete: ``false``
 
-.. _message-splitting:
 
-Message Splitting
-=================
+AMQP Queues
+===========
 
-To accomodate large messages, dripline payloads can be split into multiple chunks after they've been encoded in JSON format.  Splitting should be done so that no payloads are above some given maximum size.  Each payload chunk is sent as its own AMQP message.  All AMQP messages that are part of the same dripline message should have identical header fields and AMQP properties with the exception of the ``message-id`` field.
+Properties
+----------
 
-For an un-split message, the ``message-id`` field is a UUID string.  When splitting a message, two pieces of information should be added:  ``[UUID]/[chunk number]/[total chunks]``.  The ``chunk number`` is the index of that chunk within the reconstructed message, and the ``total chunks`` is the number of chunks into which the message has been split.
+* Passive: ``false``
+* Durable: ``false``
+* Exclusive: ``true``
+* Auto Delete: ``true``
 
 
 .. _broadcast-requests:
```

Comment:
line 75: consistency check: currently we implement set conditions badly, probably; only a subset of the providers have a real implementation, and they check if it's a condition they are looking for and otherwise ignore it; everything that is looking for that particular condition just returns (not sends an error code)